package.json

```diff
@@ -8,8 +8,12 @@
 		"fabric nodesdk"
 	],
 	"dependencies": {
+		"@types/fs-extra": "^9.0.1",
+		"@types/log4js": "^2.3.5",
+		"@types/passport": "^1.0.4",
```

Comment:
These packages should be put in devDependencies.

Reply:
Thank you. I have moved to devDependencies.